src/adapters/aave-v2/products/pool/aaveV2BasePoolAdapter.ts

```diff
@@ -31,6 +33,14 @@ type AaveV2PoolMetadata = Record<
   }
 >
 
+const protocolDataProviderContractAddresses: Partial<Record<Chain, string>> = {
+  [Chain.Ethereum]: '0x057835Ad21a177dbdd3090bB1CAE03EaCF78Fc6d',
+  [Chain.Polygon]: '0x7551b5D2763519d4e37e8B81929D336De671d46d',
+  [Chain.Avalanche]: '0x65285E9dfab318f57051ab2b139ccCf232945451',
+}
+
+const RAY = 10 ** 27
```

Comment:
According to their glossary, A ray is just a unit with 27 decimals of precision.

https://docs.aave.com/developers/v/2.0/glossary